card/lib/card/cache/persistent.rb

```diff
@@ -63,7 +63,7 @@ def annihilate
       # the cache. Note that Cardio.cache is a simple Rails::Cache, not
       # a Card::Cache object.
       def stamp
-        @stamp ||= Cardio.cache.fetch stamp_key { new_stamp }
+        @stamp ||= Cardio.cache.fetch(stamp_key) { new_stamp }
```

Comment:
believe it or not, this was not working.  Our entire stamp system was broken because of missing parentheses.  I'm amazed things weren't much worse.